src/components/m-annotator.vue

```diff
@@ -199,27 +199,32 @@ export default {
       if (this.initial_imdata.boxes != null) {
           console.log('drawing boxes', this.initial_imdata.boxes)
         for (const boxdict of this.initial_imdata.boxes) {
-            let rdict = this.rescale_box(boxdict, this.height_ratio, this.width_ratio);
```

Comment:
these seem to be unrelated changes. I'll see if I can delete it from the pull request ( you can include them later)